lib/modules/hover.js

```diff
@@ -243,6 +250,11 @@ class Hover {
 	begin() {
 		if (!this._enabled) return false;
 
+		if (this._options.closeOnTargetUnmount) {
+			const target = this.getCheckedTarget();
+			waitForRemoval(target).then(() => this.close());
```

Comment:
Since `target` is mutable, `waitForRemoval` may resolve after `target` has changed.

Reply:
What I mean here, is this hover instance can be triggered from another tag icon and mutate the `target` variable. This creates an additional observer, this time for for the new `target`. If the old `target` element then were to removed (for whatever some reason), the new hover instance would be closed.

Reply:
Okay, got it. Will a check like
```js
() => target === this.getCheckedTarget() && this.close()
```
be better?

Reply:
Yeah, that should be sufficent.